Gemfile.lock

```diff
@@ -680,4 +680,4 @@ DEPENDENCIES
   webpacker
 
 BUNDLED WITH
-   2.3.8
+   2.3.7
```

Comment:
This probably won't affect anything, but we should try not to reduce bundle versions if we don't need to.

Reply:
Good Catch. I'll patch that up